atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraContainer.java

```diff
@@ -75,9 +79,26 @@ public String getDockerComposeFile() {
 
     @Override
     public SuccessOrFailure isReady(DockerComposeRule rule) {
-        return SuccessOrFailure.onResultOf(() -> CassandraKeyValueServiceImpl.create(
+        return SuccessOrFailure.onResultOf(() -> {
+            CassandraClusterOperations cassandraOperations = new CassandraClusterOperations(rule,
+                    CASSANDRA_VERSION,
+                    ImmutableList.of(CONTAINER_NAME),
+                    CLI_CONTAINER_NAME);
+
+            if (!cassandraOperations.nodetoolShowsAllCassandraNodesUp()) {
```

Comment:
This is a change of behavior. Some tests might not require all nodes to be up and trying to enforce that here seems like a stricter check. The earlier check `createKVS` can succeed if we have quorum too. We can perhaps have a classRule in the test classes to disableAutoCompaction if thats the first thing to do? Does this nodetool command require the node to be up first?

Reply:
Note that there's only one node in the cluster, so quorum is the same as 'all' here - so in which case would `createKVS` succeed where getting nodetool status fails?

It seems to me that there would just be a race condition where nodetool status would not show all nodes as up, but in the same iteration, `createKVS` succeeds - and in this case, we'd succeed a second later anyway. Attempting to divine which classes need to have `disableAutoCompaction` set would lead to more complex code and we might not catch all cases where the tests would lead to compaction.

Reply:
Cool, lets leave it as is.